src/onefetch/cli.rs

```diff
@@ -228,6 +229,12 @@ impl Cli {
                 .takes_value(true)
                 .help("Ignore all files & directories matching EXCLUDE."),
             )
+        .arg(
+            Arg::with_name("json")
+                .short("j")
+                .long("json")
+                .help("Print out a json representation of the normal onefetch output")
+            )
```

Comment:
It might be better as `--format=json` instead of `--json`. The `--format=` option could then be reused if other output formats are supported later. If the `format` option isn't used, it could default to "human" output.

Reply:
Sounds like a reasonable proposal, will change that.